samples/nrf9160/modem_shell/README.rst

```diff
@@ -379,6 +379,18 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+DK buttons
+==========
+
+The buttons have the following functions:
+
+Button 1:
+    * Pressing of the button: raising a kill/abort signal. I.e. long enough press of the button will kill/abort all supported running commands.
+    * Releasing of the button: resetting a kill/abort signal.
+
+Button 2:
+    * Enabling / disabling the UARTs for power consumption measurements.
```

Comment:
```suggestion
    * Enabling / disabling the UARTs for power consumption measurements. Toggles between UARTs enabled and disabled.
```

Reply:
@hiltunent Can you confirm that this is about toggling UARTs between enabled and disabled?

Reply:
yes, that is correct.

Reply:
Updated.